scripts/host-check

```diff
@@ -319,21 +315,28 @@ class CheckState(enum.Enum):
     """The result state of a check."""
 
     SUCCESS = "success"
+    NEUTRAL = "neutral"
+
     WARNING = "warning"
     SKIPPED = "skipped"
     FAILED = "failed"
-    NEUTRAL = "neutral"
 
-    def is_error(self) -> bool:
+    ERROR = "error"  # Check state for when the check fails to run/errors
+
+    def is_failure(self) -> bool:
         return self in [
             CheckState.FAILED,
             CheckState.SKIPPED,
             CheckState.WARNING,
         ]
 
+    def is_success(self) -> bool:
+        return self in [CheckState.SUCCESS, CheckState.NEUTRAL]
```

Comment:
Nope, `self.is_failure()` doesn't include `Checkstate.ERROR`. I made the distinction so that we have 3 separate groupings of check states: 
1. "success" - SUCCESS and NEUTRAL
2. "failure" - FAILED, SKIPPED, WARNING
3. "error" - ERROR

Reply:
This way it's easier to write the code for handling the output message in the case of errors and failures/errors and no failures/etc.